cli/dcoscli/main.py

```diff
@@ -20,12 +22,44 @@ def main():
         return 1
 
 
+def _get_versions(config):
+    """Get DCOS and DCOS-CLI versions
+
+    :param config: config
+    :type config: Toml
+    :returns: versions of DCOS and DCOS-CLI
+    :rtype: dict
+    """
+
+    dcos_url = config.get("core.dcos_url")
+    if dcos_url:
+        url = urllib.parse.urljoin(dcos_url, 'dcos-metadata/dcos-version.json')
+        try:
+            res = http.get(url)
+            if res.status_code == 200:
+                dcos_info = res.json()
+        except Exception:
+            pass
+
+    return (
+        "dcoscli.version={}\n".format(dcoscli.version) +
+        "dcos.version={}\n".format(dcos_info.get("version", "N/A")) +
+        "dcos.commit={}\n".format(dcos_info.get(
+            "dcos-image-commit", "N/A")) +
+        "dcos.bootstrap-id={}".format(dcos_info.get("bootstrap-id", "N/A"))
+    )
```

Comment:
Okay for now. Let's file an issue to use the same code we wrote for `dcos config show`.